### PR TITLE
Add Mac screen lock menu extra

### PR DESCRIPTION
**Pull request by paulschreiber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Take the following steps to secure your devices and accounts.
 - [ ] Configure your computer to require a password after 5 minutes of inactivity
 - [ ] Configure your computer to require a password on wake
 - [ ] Learn the keyboard shortcut to lock your computer - [Windows logo + L](https://support.microsoft.com/en-us/help/12445/windows-keyboard-shortcuts) (Windows), [control + shift + power/escape](http://www.macworld.co.uk/how-to/mac/how-lock-mac-3639053/) (Mac), or [ctrl + alt + L](https://askubuntu.com/questions/126782/keyboard-shortcut-for-lockscreen-not-working) (Linux)
+- [ ] Mac: add screen lock menu extra (`open /Applications/Utilities/Keychain\ Access.app/Contents/Resources/Keychain.menu/`)
```
Comment (alulsh) [inline README.md:10]: - "Mac: add screen lock menu extra (`open /Applications/Utilities/Keychain\ Access.app/Contents/Resources/Keychain.menu/`)" --> "Mac: [add keychain status to your menu bar](http://osxdaily.com/2011/02/10/lock-mac-desktop-via-menu/) (`open /Applications/Utilities/Keychain\ Access.app/Contents/Resources/Keychain.menu/`) for easy screen locking"

**Comment by alulsh** [comment]
Thanks @paulschreiber, this is great!

I added a [quick line note](https://github.com/alulsh/personal-security-checklist/pull/1/files#r88805417). Once this is taken care of I'll gladly merge this. 

I added a link to an [OS X daily article](http://osxdaily.com/2011/02/10/lock-mac-desktop-via-menu/) (to help explain what this does for people unfamiliar with Keychain) and changed the language to match the style of the rest of the personal security checklist.

**Comment by paulschreiber** [comment]
Sounds good -- go ahead and update this using language that works for you. I'll close this PR.

**Comment by alulsh** [comment]
Done in https://github.com/alulsh/personal-security-checklist/pull/2!

**Pull request closed without merging**